### PR TITLE
Acceptance test for emberGenerateDestroy

### PR DESCRIPTION
**Pull request by trabus**
@Turbo87 so there are some issues with the new syntax not clearing eslint. I was just adding some tests, but decided to push this up to see if I could get some help figuring out what we should do to fix it.

### CODE REVIEW

**Comment by Turbo87** [comment]
you'll have to add the following snippet to the `.eslintrc` file:

```js
  parserOptions: {
    ecmaVersion: 6,
  },
```

**Comment by Turbo87** [comment]
@trabus it might make sense to also test for the negative case. i.e. if `emberGenerateDestroy()` is returning a rejecting promise if a file does not exist.

**Comment by trabus** [comment]
@Turbo87 I'm not quite sure how we should proceed to work around the no-unused-vars error. Not a lot of solutions I'd want to use here: https://github.com/eslint/eslint/issues/1939

Thoughts?

**Comment by Turbo87** [comment]
remove the variable if you only use the file callback parameter. 

eventually we should separate the blueprint into implicit/explicit API.

**Comment by trabus** [comment]
> remove the variable if you only use the file callback parameter.

But without passing file, it doesn't push to `_files` to validate.

File: tests/acceptance/helpers-test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var setupTestHooks = require('../../lib/helpers/setup');
+
+var blueprintHelpers = require('../../helpers');
+var emberNew = blueprintHelpers.emberNew;
+var emberGenerateDestroy = blueprintHelpers.emberGenerateDestroy;
+
+var chai = require('../../chai');
+var expect = chai.expect;
+var file = chai.file;
```
Comment (Turbo87) [inline tests/acceptance/helpers-test.js]: I'm talking about removing this line.

Comment (trabus) [inline tests/acceptance/helpers-test.js]: Ah, duh. Thanks.

**Comment by Turbo87** [comment]
@homu r+

**Comment by trabus** [comment]
Oh, I was going to add more tests. Will do on a new branch. :)
